### PR TITLE
set history depth

### PR DESCRIPTION
**Pull request by Karsten1987**
Given that https://github.com/ros-visualization/rqt_publisher/issues/10 was closed, I've opened this PR against `crystal-devel`. 

Signed-off-by: Karsten Knese <karsten@openrobotics.org>

### CODE REVIEW

**Comment by dirk-thomas** [comment]
The current patch breaks Crystal. Based on ros2/rclpy#362 please update the patch to pass an explicit QoS profile as a keyword argument.

**Comment by Karsten1987** [comment]
done in https://github.com/ros-visualization/rqt_publisher/pull/12/commits/3175722789f66831e555e4a1fefe48834cbaf8a2

File: src/rqt_publisher/publisher.py [hunk 1]
```diff
@@ -137,7 +138,7 @@ def _add_publisher(self, publisher_info):
 
         # create publisher and timer
         publisher_info['publisher'] = self._node.create_publisher(
-            msg_module, publisher_info['topic_name'])
+            msg_module, publisher_info['topic_name'], QoSProfile(depth=10))
```
Comment (dirk-thomas) [inline src/rqt_publisher/publisher.py]: This profile needs to be passed as a keyword argument in order to be compatible with Crystal.

Comment (Karsten1987) [reply src/rqt_publisher/publisher.py]: done in 
4a48baf

**Pull request merged**